content/en/agent/logs/proxy.md

```diff
@@ -42,12 +41,32 @@ The parameters above can also be set with the following environment variables:
 * Then configure your proxy to listen on `<PROXY_PORT>` and forward the received logs. For {{< region-param key="dd_subdomain" code="true" >}}.{{< region-param key="dd_site" code="true" >}}, use {{< region-param key="tcp_endpoint" code="true" >}} on port {{< region-param key="tcp_endpoint_port" code="true" >}} and activate SSL encryption.
```

Comment:
@ruthnaebeck Hi there! Any recommendations on how to fix this area? It looks like nothing is populating for the region params.

Reply:
Updated :)